octodns_ultra/__init__.py

```diff
@@ -325,6 +328,19 @@ def populate(self, zone, target=False, lenient=False):
                 name = zone.hostname_from_fqdn(record['ownerName'])
                 if record['rrtype'] == 'SOA (6)':
                     continue
+                if (
+                    record['rrtype'] == 'A (1)'
+                    and 'profile' in record
+                    and record['profile']['@context']
+                    == 'http://schemas.ultradns.com/DirPool.jsonschema'
+                ):
+                    self.log.warning(
+                        'populate: ignoring record with '
+                        'unsupported type, %s  %s',
+                        name,
+                        '[Directional Pool (Dynamic)]',
+                    )
```

Comment:
Since the 2nd param is a static string it makes sense to just include it in the main str rather than have a placeholder.

```suggestion
                    self.log.warning(
                        'populate: ignoring record with unsupported type, %s [Directional Pool (Dynamic)]',
                        name,
                    )
```

Reply:
Makes perfect sense, I was just following the other section and didn't think.